subsys/bluetooth/controller/hci/hci.c

```diff
@@ -214,6 +215,7 @@ static void reset(struct net_buf *buf, struct net_buf **evt)
 	hci_hbuf_total = 0;
 	hci_hbuf_sent = 0;
 	hci_hbuf_acked = 0;
+	memset(hci_hbuf_pend, 0x00, sizeof(hci_hbuf_pend));
```

Comment:
I think our general style convention is to use just 0 (instead of 0x00) when setting memory to zero.